templates/frontend/pages/userLostPassword.tpl

```diff
@@ -26,7 +26,8 @@
 			</div>
 		{/if}
 
-		<fieldset class="fields">
+		<div class="fields">
+			<legend class="pkp_screen_reader">{translate key="user.email"}</legend>
```

Comment:
Because this has changed from a `<fieldset>` to a `<div>`, you can remove the `<legend>`. I think this form is only one field so doesn't need the `<fieldset>`.